static/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@systimotic here is the `manifest.json`. All the gatsby plugin does it generate this file. You still have to manually input the config into the plugin options